opm/simulators/wells/StandardWell_impl.hpp

```diff
@@ -976,6 +976,8 @@ namespace Opm
             if ( (this->isProducer() && pressure_diff < 0.) || (this->isInjector() && pressure_diff > 0.) ) {
                 deferred_logger.debug("CROSSFLOW_IPR",
                                 "cross flow found when updateIPR for well " + name());
+                // we ignore these connections for now
+                continue;
```

Comment:
Can you explain a little more here also about the exact purpose?

And also, a message here (warning or debug) telling that the crossflow is ignored here.

Reply:
I think the IPR formulation handle crossflow at the moment. I guess that is the reason for giving a debug output here.. But since the formulation do not handle crossflow adding these connections may lead to premature closing of wells. 
I can update the debug message with " ... The connection is ignored." or something similar.